### ISSUE TITLE
👍

### ISSUE THREAD

**Issue by NWPUZhoufei** [YingYuChen97/CSCA#1]

**Comment by NWPUZhoufei** [comment]
> Hello！ I have two questions:
> 
> 1. From your description, you are doing it on the feature of PT-MAP. Your method doesn't seem to rely on shrewdly designed training methods, it seems that normal transfer learning pre-training will work. So, is your method able to achieve the reported performance on standard supervised classification pretrained models?
> 2. Why do you need to perform feature extraction on the test set images in advance? As far as I understand, when using the pre-trained feature extractor to perform feature extraction on the images of the entire dataset, it is equivalent to linking to the query data, such as when performing BN. So, can your method achieve comparable performance without pre-extracting features?
>    Thank you！
